controllers/members.js

```diff
@@ -46,7 +49,30 @@ const getIdleMembers = async (req, res) => {
   }
 }
 
+/**
+ * Makes a new member a member
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const moveToMembers = async (req, res) => {
+  try {
+    const username = req.params.username
+    const result = await fetchUser({ username: username })
```

Comment:
We can use JS shorthand here
https://alligator.io/js/object-property-shorthand-es6/

Would become `fetchUser({ username })` then